### ISSUE TITLE
Problem z wersją 1.0 - captcha?

### ISSUE THREAD

**Issue by ddand** [MK-PL/AllegroRSS#29]
Cześć, na wstępie bardzo dziękuję za długo oczekiwaną aktualizację skryptu!!

Wczoraj wszystko pięknie działało, ale niestety dziś pojawił się problem - wydaje mi się, że przy aktualizacji kanałów Allegro żąda rozwiązania captchy i to powoduje zamknięcie aplikacji node.js. Może to wynika, ze zbyt wielu zapytań z jednego hosta? Mój czytnik RSS przy uruchomieniu aktualizuje wszystkie kanały (mam ich ok. 20). Niestety okna tak szybko znikają, że nie jestem w stanie przeczytać, jaki jest dokładnie komunikat.

**Comment by ddand** [comment]
Może rozwiązaniem tymczasowym było by dodanie pauzy przy pojawieniu się żądania rozwiązania captchy? W tym momencie skrypt praktycznie nie działa.

**Comment by ddand** [comment]
Tymczasowe rozwiązanie to czytnik RSS, w który można zdefiniować różne czasy aktualizacji i ograniczyć liczbę żądań połączenia do 1. Póki co, działa.

**Comment by MK-PL** [comment]
@ddand tak, to jest zabezpieczenie przeciwko wysyłaniu zbyt wielu zapytań naraz. Na ten moment mogę tylko zaproponować ograniczenie częstotliwości sprawdzania kanałów RSS w czytniku.

**Comment by ddand** [comment]
A może skrypt używający głównego profilu Chrome z zalogowanym użytkownikiem byłby bardziej odporny na blokowanie? Mimo znacznego ograniczenia liczby i częstotliwości zapytań raz/dwa razy na dobę pojawia się captcha. Mam też wrażenie, że te tymczasowe instancje Chrome'a zapisują bardzo dużo danych w katalogu Windows\Temp (win10), ale nie jestem pewien, czy to efekt działania AllegroRSS.
